<?xml version="1.0"?>

<launch>
  <arg name="frame"/>
  <arg name="worldFrame" default="world"/>
<!-->  <arg name="x2"/> <!-->
  <node name="controller" pkg="crazyflie_controller" type="crazyflie_controller" output="screen">
    <param name="frame" value="$(arg frame)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
	<!-->  editted <!-->
<!-->    <param name="x2" value="$(arg x2)" />  <!-->
    <rosparam command="load" file="$(find crazyflie_controller)/config/crazyflie2.yaml" />
  </node>
</launch>
